### PR TITLE
Feature/fbar 131 filters markup

### PR DESCRIPTION
**Pull request by paulmskim**
**Ticket:** [FBAR-143]

Requires: https://github.com/moderntribe/tribe-common/pull/1431
This uses the `border-small` class in the "Today" button and adds border button styles to the customizer.

**Screenshot:**
image

[FBAR-143]: https://moderntribe.atlassian.net/browse/FBAR-143

### CODE REVIEW

File: src/Tribe/Customizer/Global_Elements.php [hunk 1]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
 				}
 			';
 
+			// overrides for common components/full/buttons/_border.pcss.
+			$template .= '
+				.tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt),
+				.tribe-common a.tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt) {
+					border-color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:89]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: e36960144e1dd221e8213538c6f61833</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 2]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
...
+					color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:90]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 27801bc49daee7a89f2249e0cccd677c</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 3]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
...
+				}
+			';
+
+			$template .= '
+				.tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):focus,
+				.tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):hover,
+				.tribe-common a.tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):focus,
+				.tribe-common a.tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):hover {
+					color: ' . $color_background . ';
+					background-color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:100]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: 59bc921c576274a8d2c149da22e57c0f</sub>

File: src/Tribe/Customizer/Global_Elements.php [hunk 4]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
...
+				}
+			';
+
+			$template .= '
+				.tribe-theme-twentyseventeen .tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):focus,
+				.tribe-theme-twentyseventeen .tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):hover,
+					background-color: <%= global_elements.accent_color %>;
```
Comment (tr1b0t) [inline src/Tribe/Customizer/Global_Elements.php:107]: :warning: Found Underscore.js unescaped output notation: "<%=".

<sub>hash: b3be6a05eb219e27a4596c24386cd15f</sub>

**Comment by paulmskim** [comment]
on hold as tests are failing

File: src/Tribe/Customizer/Global_Elements.php [hunk 5]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
 				}
 			';
 
+			// overrides for common components/full/buttons/_border.pcss.
```
Comment (lucatume) [inline src/Tribe/Customizer/Global_Elements.php:85]:
```suggestion
			// Overrides for common components/full/buttons/_border.pcss.
```

**Review by lucatume — APPROVED** [review]

File: src/Tribe/Customizer/Global_Elements.php (hunk 2 above)
Comment (Camwyn) [reply src/Tribe/Customizer/Global_Elements.php:90]: @paulmskim is this something we should account for in our linters?

File: src/Tribe/Customizer/Global_Elements.php [hunk 6]
```diff
@@ -82,6 +82,32 @@ public function get_css_template( $template ) {
...
+			';
+
+			$template .= '
+				.tribe-theme-twentyseventeen .tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):focus,
+				.tribe-theme-twentyseventeen .tribe-common .tribe-common-c-btn-border:not(.tribe-common-c-btn-border--secondary):not(.tribe-common-c-btn-border--alt):hover,
+					background-color: <%= global_elements.accent_color %>;
+				}
+			';
+
 			// overrides for common components/full/buttons/_solid.pcss.
```
Comment (Camwyn) [inline src/Tribe/Customizer/Global_Elements.php:111]:
```suggestion
			// Overrides for common components/full/buttons/_solid.pcss.
```

Comment (Camwyn) [reply src/Tribe/Customizer/Global_Elements.php:111]: Sorry, missed this on first pass 🤦

**Pull request merged**